packages/docs/getting-started/running-a-validator.md

```diff
@@ -135,6 +137,15 @@ export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:baklava
 export NETWORK_ID=1101
 ```
 
+For having the Proxy and the Validator reporting to the network EthStats service, you can export the following 2 environment variables:
+
+```bash
+export VALIDATOR_ETHSTATS_FLAG="--ethstats=PROXY-NAME@baklava-ethstats.celo-testnet.org"
+export PROXY_ETHSTATS_FLAG="--ethstats=VALIDATOR-NAME@baklava-ethstats.celo-testnet.org"
+```
+
+If you set the above environment variables, you could see your Proxy and Validator reporting in the [network Monitoring Tool](https://baklava-ethstats.celo-testnet.org/).
```

Comment:
nit: "By setting the above environment variables you will be able to see your Proxy and Validator in the network monitoring tool after you're done setting them up."